<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Segmentation Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="GLSegmentEditor" name="gleditor" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>710</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTrees">
    <property name="title">
     <string>Trees</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExport_Points"/>
    <addaction name="separator"/>
    <addaction name="actionExport_To_Database"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTrees"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExport_Points">
   <property name="text">
    <string>Export Points</string>
   </property>
  </action>
  <action name="actionExport_To_Database">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export To Database</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GLSegmentEditor</class>
   <extends>QWidget</extends>
   <header>.glsegmenteditor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
